src/monster.cpp

```diff
@@ -932,7 +926,7 @@ void Monster::onThinkDefense(uint32_t interval)
 		}
 	}
 
-	if (!isSummon() && summons.size() < mType->maxSummons) {
+	if (!isSummon() && summons.size() < mType->maxSummons && hasFollowPath) {
```

Comment:
Can you link me to the issues, so we can speed up the release of 1.2. I would love to take a look at it.